build_darwin.sh

```diff
@@ -19,13 +19,13 @@ echo "965cc5a8bb46ce4199a47e9b2c9e1cae3b137e8356ffdad6d94d3b9069b71dc2  libevent
 tar -zxvf libevent-$LIBEVENT_VERSION.tar.gz && \
 cd libevent-$LIBEVENT_VERSION && \
 ./configure \
-            LDFLAGS="-L$PWD/../openssl-$OPENSSL_VERSION/" \
-            CPPFLAGS="-I$PWD/../openssl-$OPENSSL_VERSION/include" \
+            LDFLAGS="-L/usr/local/opt/openssl/lib" \
+            CPPFLAGS="-I/usr/local/opt/openssl/include" \
```

Comment:
Why are you using the system-wide /usr/local/opt version instead of the local newly built version?